sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -1694,6 +1747,40 @@ public partial class CassandraTableResourceInfo
         public Azure.ResourceManager.CosmosDB.Models.CassandraSchema Schema { get { throw null; } set { } }
         public string TableName { get { throw null; } set { } }
     }
+    public partial class ClientEncryptionIncludedPath
+    {
+        public ClientEncryptionIncludedPath(string path, string clientEncryptionKeyId, string encryptionType, string encryptionAlgorithm) { }
+        public string ClientEncryptionKeyId { get { throw null; } set { } }
+        public string EncryptionAlgorithm { get { throw null; } set { } }
+        public string EncryptionType { get { throw null; } set { } }
+        public string Path { get { throw null; } set { } }
+    }
+    public partial class ClientEncryptionKeyGetPropertiesResource : Azure.ResourceManager.CosmosDB.Models.ClientEncryptionKeyResource
+    {
+        public ClientEncryptionKeyGetPropertiesResource() { }
+        public Azure.ETag? ETag { get { throw null; } }
+        public string Rid { get { throw null; } }
+        public float? Timestamp { get { throw null; } }
+    }
+    public partial class ClientEncryptionKeyGetResultCreateOrUpdateContent
+    {
+        public ClientEncryptionKeyGetResultCreateOrUpdateContent(Azure.ResourceManager.CosmosDB.Models.ClientEncryptionKeyResource resource) { }
+        public Azure.ResourceManager.CosmosDB.Models.ClientEncryptionKeyResource Resource { get { throw null; } }
+    }
+    public partial class ClientEncryptionKeyResource
```

Comment:
Shall we rename this class to `CosmosDBSqlDatabaseClientEncryptionKeyBaseProperties`?
`Resource` is kind of reserved suffix in mgmt dotnet SDK, only class that inherits from `ArmResource` could end with this suffix (which is effectively resource clients), this class is purely a data model

Reply:
sure.

Reply:
is now CosmosDBSqlClientEncryptionKeyResource. Similar to  SqlContainerResource

Reply:
Does using rename-model: suffice. Do have a look at the changes made.

Reply:
#resolved